echopype/tests/calibrate/test_calibrate.py

```diff
@@ -154,13 +154,12 @@ def check_output(base_path, ds_cmp, cal_type):
         }
         for fidx in range(4):  # loop through all freq
             assert np.alltrue(
-                ds_cmp.echo_range.isel(frequency=fidx).values
+                ds_cmp.echo_range.isel(frequency=fidx, ping_time=0).values[None, :]
```

Comment:
Oh I didn't know you can do this to have equivalent results as `np.expand_dims`!

I wonder which one is more readable or pythonic, but it probably doesn't matter -- this looping is annoying because of how loadmat gives as outputs, but perhaps a better way to do this is to produce something from the loadmat results that can be compared without the loops. Something for laterrrr when we update and clean up the tests!

Reply:
Yeah, I guess you could use `np.expand_dims`. I am fine with using either one. If there is a strong push for `np.expand_dims` let me know and I can change it. If not, we can save this for later as you suggested.

Reply:
Let's save this for later. Feel free to self merge!